pkg/components/imguploader/webdavuploader.go

```diff
@@ -53,13 +55,18 @@ func (u *WebdavUploader) Upload(pa string) (string, error) {
 		return "", fmt.Errorf("Failed to upload image. Returned statuscode %v body %s", res.StatusCode, body)
 	}
 
-	return url.String(), nil
+	if u.public_url != "" {
+		return (u.public_url + filename), nil
```

Comment:
Well i planned initially to use path concatenation, but this always adds a /. This doesn't work out for GET arguments. Do you have any suggestion ?